packages/app/src/components/pages/HomePage/HomePage.tsx

```diff
@@ -1,22 +1,44 @@
 import React from 'react';
 import { createUseStyles } from 'react-jss';
-import { Table } from '@cardz/components';
+import { Table, addRandomCardToTable, addRandomCardDeckToTable } from '@cardz/components';
```

Comment:
I haven't done this part, I feel like we could create a ticket for that, I didn't want to spend too much time on this in this PR since it is not critical 😄